scripts/check_platform_docs.py

```
@@ -240,6 +260,26 @@ def role_faults():
 
 
+def prefix_faults(rows):
+    """Every header a platform declares begins with SolidSyslog<Token>.
+
+    docs/NAMING.md states this for public classes. An *Errors.h takes its name
+    from the class it belongs to, so it carries the token by construction and is
+    checked alongside the rest — one that does not is an orphan worth catching.
+    """
+    faults = []
+    for token, directory in rows:
+        accepted = [token] + CLASS_PREFIXES.get(token, [])
```

Ruff (0.16.1)

[warning] Consider `[token, *CLASS_PREFIXES.get(token, [])]` instead of concatenation

Replace with `[token, *CLASS_PREFIXES.get(token, [])]`

(RUF005)